vocabularies/sample-type.ttl

```diff
@@ -291,12 +311,19 @@ sampty:polished-block a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
 sampty:powdered-chip a skos:Concept ;
-    skos:definition "A rock chip sample that has subsequently been powdered"@en ;
+    skos:definition "A rock chip sample that has subsequently been powdered."@en ;
     skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
     skos:notation "PCHIP" ;
     skos:prefLabel "Powdered Chip"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:pulp a skos:Concept ;

```

Comment:
I have never heard of the term pulp in geology. Usually crushing, grinding and smiling are referred to for creating powdered samples.

Reply:
It's a mineral geochemistry thing. I'm still unsure how Pulps are different to powered rock samples, but the Minerals people assure me there is a difference.

Reply:
All good. Derek has commented that's its a term he is familiar with, so I'm good with it.

tech good.